src/fswAlgorithms/sensorInterfaces/scanningInstrumentController/_UnitTest/test_scanningInstrumentController.py

```diff
@@ -0,0 +1,152 @@
+# 
+#  ISC License
+# 
+#  Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado Boulder
+# 
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+# 
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+# 
+# 
+
+#
+#   Unit Test Script
+#   Module Name:        scanningInstrumentController
+#   Author:             Lorenzzo Mantovani
+#   Creation Date:      September 11th, 2023
+#
+
+import pytest
+import os
+import inspect
+
+from Basilisk.utilities import SimulationBaseClass
+from Basilisk.utilities import unitTestSupport
+from Basilisk.architecture import messaging
+from Basilisk.utilities import macros
+from Basilisk.fswAlgorithms import scanningInstrumentController
+from Basilisk.architecture import bskLogging
+
+filename = inspect.getframeinfo(inspect.currentframe()).filename
```

Comment:
Are you using Pycharm or some similar Python IDE with a linter conigured? If so I won't mention an of the Python formalities as I assume your IDE is already barking at you. 👍

Reply:
I configured linter properly now. Code should be adequate now.